From 9752b2bfe5409d83d60743e3bb9fbc5765e4d15b Mon Sep 17 00:00:00 2001
From: Wei Xu <weixugc@google.com>
Date: Fri, 28 Feb 2020 11:12:12 -0800
Subject: [PATCH] Add 'Google' to supported platform of DataSourceGCE.

This adds "Google" as a supported platform of DataSourceGCE, in
addition to "Google Compute Engine".
---
 cloudinit/sources/DataSourceGCE.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cloudinit/sources/DataSourceGCE.py b/cloudinit/sources/DataSourceGCE.py
index 7fb43c09..230ee51d 100644
--- a/cloudinit/sources/DataSourceGCE.py
+++ b/cloudinit/sources/DataSourceGCE.py
@@ -254,7 +254,7 @@ def read_md(address=None, platform_check=True):
 
 def platform_reports_gce():
     pname = util.read_dmi_data('system-product-name') or "N/A"
-    if pname == "Google Compute Engine":
+    if pname == "Google Compute Engine" or pname == "Google":
         return True
 
     # system-product-name is not always guaranteed (LP: #1674861)
-- 
2.25.1.481.gfbce0eb801-goog

